sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -30,6 +30,14 @@ Use the client library for App Configuration to create and manage application co
 All client libraries, by default, use Netty HTTP client. Adding the above dependency will automatically configure 
 AppConfiguration to use Netty HTTP client. 
 
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-core-http-netty</artifactId>
+    <version>1.0.0-preview.5</version>
```

Comment:
Core http netty version will be preview.7 after I merge this https://github.com/Azure/azure-sdk-for-java/pull/5777